src/test/kotlin/coverosR3z/timerecording/UITimeEntryTests.kt

```diff
@@ -254,6 +279,27 @@ class UITimeEntryTests {
         pom.etp.enterTime(project, "1", "", dateString)
     }
 
+
+    private fun calcDateString(date : Date) : String {
+        return if (pom.driver is ChromeDriver) {
+            date.chromeStringValue // returns the chrome format
+        } else {
+            date.stringValue // returns the chrome format
+        }
+    }
+
+
+    private fun enterThreeEntriesForEmployee(project: String) {
+        val date = Date(2021, Month.JAN, 1)
```

Comment:
```suggestion
        // heads up: this will start failing when we create a concept of `time period`
        val date = Date(2021, Month.JAN, 1)
```